java/server/src/org/openqa/selenium/grid/distributor/remote/RemoteDistributor.java

```diff
@@ -87,6 +87,18 @@ public RemoteDistributor add(Node node) {
     return this;
   }
 
+  @Override
+  public void drain(UUID nodeId) {
+    Objects.requireNonNull(nodeId, "Node ID must be set");
```

Comment:
```suggestion
    Require.nonNull("Node ID must be set", nodeId);
```

Since we use `Require.nonNull()` everywhere.